dev-packages/private-re-exports/src/package-re-exports.ts

```diff
@@ -17,8 +17,7 @@
 import cp = require('child_process');
 import fs = require('fs');
 import path = require('path');
-import { parseModule } from '.';
-import { PackageJson, ReExportJson } from './utility';
```

Comment:
@tsmaeder if by drive-by changes you mean ones like this, this is actually to prevent some circular dependencies that prevented me from building.

Reply:
Those imports are ticking bombs waiting to blow up in someone's face.

As it blew up in mine, I didn't try to untangle the cyclic import error and just nuked the issue.

I'll try to make a PR just for that if that helps.